examples/php-features-test/test-all.sh

```
@@ -0,0 +1,262 @@
+        laravel) output_dir="gen/php/laravel" ;;
+        symfony) output_dir="gen/php/symfony" ;;
+        async) output_dir="gen/php/async" ;;
+        full) output_dir="gen/php/full" ;;
+        clientOnly) output_dir="gen/php/client" ;;
+    esac
+    
+    if [ -d "$output_dir" ]; then
+        print_success "Output directory created: $output_dir"
+        
+        # Count generated files
+        local file_count=$(find "$output_dir" -name "*.php" | wc -l)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)